src/QuadForm/Torsion.jl

```diff
@@ -1751,6 +1771,11 @@ function genus(T::TorQuadModule, signature_pair::Tuple{Int, Int})
   end
   # This genus has the right discriminant group
   # but it may be empty
+  #
+  # in this case the blocks of scales 1, 2, 4 are under determined
```

Comment:
This part of the comment still belongs to the odd case where modulus_quadratic_form(T) == 1.
The following two lines can stay.